.github/workflows/docker-GA-images.yml

```diff
@@ -153,23 +99,13 @@ jobs:
           file: ./packaging/src/docker/Dockerfile
           platforms: linux/amd64,linux/arm64
           push: true
-          tags: apache/hive:${{ env.tag }}
+          tags: ${{ env.namespace }}/hive:${{ env.tag }}
```

Comment:
Please check this,we i implemented this last time ` ${{ env.namespace }}` ended up returning the wrong value like `apachejenkins` or something like that and the docker image was  pushed to wrong repo. As a result, i had to hard code it.

Reply:
Could you elaborate on that issue? When each contributor tests it on their own repo, when the configuration is wrong, pushing an image would fail with an authorization error like [this](https://github.com/okumin/hive/actions/runs/18624155550). In case that a release manager builds it in apache/hive, [the apache repo is correctly used](https://github.com/apache/hive/blob/5f91fa9c23a7ce8f6ccf5df8171cf0695f917900/.github/workflows/docker-GA-images.yml#L72).

Reply:
This is what i was referring to (found the exact jira):
 https://issues.apache.org/jira/browse/HIVE-27277?focusedCommentId=17759033&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17759033 

PR: https://github.com/apache/hive/commit/474c2e27e0b9715e3065a7f1a84cc1186f67d2ed


So i was using `tags: ${{ secrets.DOCKERHUB_USER }}/hive:${{ env.tag }}`  secrets.DOCKERHUB_USER should ideally be the repo name which is "apache" but it turns out its "afsjenkins" when building. 


So just wanted to make sure we are not running into a similar  issue again. 

Apart for this, rest of the PR LGTM +1